README.adoc

```diff
@@ -114,19 +114,19 @@ Spring Boot can configure large parts of typical applications based on the conte
 A single `@EnableAutoConfiguration` annotation triggers auto-configuration of the Spring context.
 
 Auto-configuration attempts to deduce which beans a user might need. For example, if `HSQLDB` is on the classpath, and the user has not configured any database connections, then they probably want an in-memory database to be defined.
-Auto-configuration will always back away as the user starts to define their own beans.
```

Comment:
I'm not sure why own is being removed here.

Reply:
"Own" is redundant in this case as we are already saying users to  _define their beans_, so there is no need for "own" according to me. Rest the decision is up to you.